lib/taskana-core/src/test/java/acceptance/task/CompleteTaskAccTest.java

```diff
@@ -73,26 +115,26 @@ void should_ForceCompleteTask_When_NoExplicitPermissionsButUserIsInAdministrativ
 
   @WithAccessId(user = "user-1-1")
   @Test
-  void testCompleteTaskTwice() throws Exception {
-    Task completedTask = taskService.completeTask("TKI:000000000000000000000000000000000002");
-    Task completedTask2 = taskService.completeTask("TKI:000000000000000000000000000000000002");
+  void should_CompleteTaskTwice() throws Exception {
+    Task task = createTaskClaimedByUser();
+
+    Task completedTask = taskService.completeTask(task.getId());
+    Task completedTask2 = taskService.completeTask(task.getId());
+
     assertThat(completedTask2).isEqualTo(completedTask);
   }
 
   @WithAccessId(user = "user-1-1")
   @Test
-  void testForceCompleteAlreadyClaimed() throws Exception {
-    Task newTask = taskService.newTask("USER-1-1", "DOMAIN_A");
-    newTask.setClassificationKey("T2100");
-    newTask.setOwner("other");
-    newTask.setPrimaryObjRef(
-        createObjectReference("COMPANY_A", "SYSTEM_A", "INSTANCE_A", "VNR", "1234567"));
-    TaskImpl newTaskImpl = (TaskImpl) newTask;
-    newTaskImpl.setState(TaskState.CLAIMED);
-    newTaskImpl.setClaimed(Instant.now());
+  void should_ForceCompleteClaimedTask_When_TaskCallerIsNotTheOwner() throws Exception {
+    Task task =
```

Comment:
Idea: change the return type of `createTaskClaimedByUser` to `TaskBuilder`.
Then you can just write Task claimedBydifferentUser ?= createTaskClaimedByUser().owner("other").buildAndStore([...])

This means that all other calls have to be `Task claimedTask = createTaskClaimedByUser().build()` . What do you think?